docs/root/intro/arch_overview/upstream/load_balancing/degraded.rst

```diff
@@ -6,12 +6,14 @@ Degraded endpoints
 Envoy supports marking certain endpoints as degraded, meaning that they are able to receive
 traffic, but should only receive traffic once there are not sufficient healthy hosts available.
 
-Routing to degraded hosts can be thought of as routing to hosts in a lower 
-:ref:`priority <arch_overview_load_balancing_priority_levels>`. As hosts in higher priorities go 
-unhealthy, traffic spills down to lower priorities. As the amount of healthy hosts
-available is no longer sufficient to handle 100% of the load, traffic will spill over to degraded 
-hosts using the same mechanism as priority spillover for healthy hosts. This ensures that 
-traffic is gradually shifted to degraded hosts as it becomes necessary.
+Routing to degraded hosts can be thought of as similar to routing to hosts in a
+lower :ref:`priority <arch_overview_load_balancing_priority_levels>`, although
+degraded hosts will count against their original priority's health percentage
+for the purposes of computing traffic spillover. As the amount of healthy hosts
```

Comment:
Maybe point to the table entry that demonstrates this? I think an example would be nice, but if its covered in the table that's probably good enough